src/libs/client.js

```diff
@@ -56,6 +57,9 @@ export const getClient = async () => {
   })
   listenTokenRefresh(client)
   client.getStackClient().setOAuthOptions(oauthOptions)
+
+  startListening(client)
```

Comment:
Here you are in `getClient` which is the method called when retrieving the client from asyncStorage on app re-start

On login this method is not called, instead we use `createClient` and `finalizeClientCreation`. So you should add this to one of them

Reply:
- [ ] Updated